### PR TITLE
Add `New Featured Artist: Rameses B` news post

### PR DESCRIPTION
**Pull request by pishifat**
🅱️

### CODE REVIEW

File: news/2022-05-28-new-featured-artist-rameses-b.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+Here's the general idea: every month, we host one (or more) playlists using Featured Artist songs and reward participants with *something*. There isn't a strict format to the playlists — the mod choices, the map selection process, and even the prizes may change month by month.
+
+We'll keep things simple for this first iteration. This month's playlists are...
+
+  - Open for **1 month**
+  - Available for **all game modes**
+  - Highlighting **8 maps** from recent Featured Artists
+  - Locked into the **Hidden mod** (with *No Fail* and *Mirror* freemod options)
+
+The **highest ranked 3 players** in each playlist will receive one of these Featured Artist themed profile badges!
```
Comment (Walavouchey) [inline news/2022-05-28-new-featured-artist-rameses-b.md:133]: this is where you'd use an en dash

```suggestion
The **highest ranked 3 players** in each playlist will receive one of these Featured Artist–themed profile badges!
```

File: news/2022-05-28-new-featured-artist-rameses-b.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+![Featured Artist profile badge](/wiki/shared/news/2022-05-28-new-featured-artist-rameses-b/faplosu.png) ![Featured Artist profile badge](/wiki/shared/news/2022-05-28-new-featured-artist-rameses-b/fapltaiko.png)
+
+![Featured Artist profile badge](/wiki/shared/news/2022-05-28-new-featured-artist-rameses-b/faplcatch.png) ![Featured Artist profile badge](/wiki/shared/news/2022-05-28-new-featured-artist-rameses-b/faplmania.png)
+
+### How to participate
+
+Not sure what *lazer* or *playlists* are? [Catch up here.](https://www.youtube.com/watch?v=ut_SOreClcc)
```
Comment (peppy) [inline news/2022-05-28-new-featured-artist-rameses-b.md:141]:
```suggestion
Not sure what *lazer* or *playlists* are? Catch up [here](https://www.youtube.com/watch?v=ut_SOreClcc).
```

File: news/2022-05-28-new-featured-artist-rameses-b.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    <source src="https://assets.ppy.sh/artists/270/Transformations/Rameses%20B%20-%20Transformations%20(feat.%20Laura%20Brehm).mp3" type="audio/mpeg">
+</audio>
+
+---
+
+That's a lot of songs, we know. And there's even more on [**Rameses B**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/270) waiting for you to begin your mapping journey. Don't keep them waiting.
+
+## Featured Artists in lazer
+
+Speaking of awesome new things, we're also experimenting with **Featured Artists in lazer playlists** starting TODAY.
```
Comment (peppy) [inline news/2022-05-28-new-featured-artist-rameses-b.md:122]:
```suggestion
Speaking of awesome new things, we're also experimenting with **Featured Artists Playlists** in osu!(lazer) starting TODAY.
```

Probably reads better like this? I forget what we agreed on for mentions of lazer in news posts so might be worth checking previous usage.

File: news/2022-05-28-new-featured-artist-rameses-b.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+![Featured Artist profile badge](/wiki/shared/news/2022-05-28-new-featured-artist-rameses-b/faplosu.png) ![Featured Artist profile badge](/wiki/shared/news/2022-05-28-new-featured-artist-rameses-b/fapltaiko.png)
+
+![Featured Artist profile badge](/wiki/shared/news/2022-05-28-new-featured-artist-rameses-b/faplcatch.png) ![Featured Artist profile badge](/wiki/shared/news/2022-05-28-new-featured-artist-rameses-b/faplmania.png)
+
+### How to participate
+
+Not sure what *lazer* or *playlists* are? [Catch up here.](https://www.youtube.com/watch?v=ut_SOreClcc)
+
+If you want to try out these new Featured Artist playlists, start by downloading lazer from its [GitHub releases page](https://github.com/ppy/osu/releases/latest). From lazer's main menu, enter the **playlists** section and filter the lobbies to **Featured Artists** in the top right corner. Enter a lobby, play each map, then you'll see your name on the leaderboards!
```
Comment (Walavouchey) [inline news/2022-05-28-new-featured-artist-rameses-b.md:143]:
```suggestion
If you want to try out these new Featured Artist playlists, start by downloading lazer from its [GitHub releases page](https://github.com/ppy/osu/releases/latest). From lazer's main menu, enter the **playlists** section and filter the lobbies to **Featured Artists** in the top-right corner. Enter a lobby, play each map, then you'll see your name on the leaderboards!
```

File: news/2022-05-28-new-featured-artist-rameses-b.md (hunk 3 above)
Comment (Walavouchey) [reply news/2022-05-28-new-featured-artist-rameses-b.md:122]: I'd rather have "Featured Artist playlists"

Comment (peppy) [reply news/2022-05-28-new-featured-artist-rameses-b.md:122]: sure

File: news/2022-05-28-new-featured-artist-rameses-b.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+    <source src="https://assets.ppy.sh/artists/270/Transformations/Rameses%20B%20-%20Transformations%20(feat.%20Laura%20Brehm).mp3" type="audio/mpeg">
+</audio>
+
+---
+
+That's a lot of songs, we know. And there's even more on [**Rameses B**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/270) waiting for you to begin your mapping journey. Don't keep them waiting.
+
+## Featured Artists in osu!(lazer)
+
+Speaking of awesome new things, we're also experimenting with **Featured Artists playlists** starting TODAY.
```
Comment (Walavouchey) [inline news/2022-05-28-new-featured-artist-rameses-b.md:122]: it's the same with "Featured Artist listing/maps/songs/etc."

```suggestion
Speaking of awesome new things, we're also experimenting with **Featured Artist playlists** starting TODAY.
```

**Review by peppy — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**